CDTDatastore/query/CDTQIndexManager.h

```diff
@@ -75,47 +77,49 @@ typedef NS_ENUM(NSInteger, CDTQQueryError) {
 /**
  Constructs a new CDTQIndexManager which indexes documents in `datastore`
  */
-+ (CDTQIndexManager *)managerUsingDatastore:(CDTDatastore *)datastore
-                                      error:(NSError *__autoreleasing *)error;
++ (nullable CDTQIndexManager *)
+managerUsingDatastore:(CDTDatastore *)datastore
+                error:(NSError *__autoreleasing __nullable *__nullable)error;
```

Comment:
although it may not be needed to tell the compiler that it this parameter can be `nil` I think its better practise to be explicit so when we are programming error handling code, it's easier for us to see that we should do the check to make sure it isn't `nil`